aten/src/ATen/core/dispatch/OperatorEntry.h

```diff
@@ -205,12 +206,16 @@ class TORCH_API OperatorEntry final {
   bool hasKernelForAnyDispatchKey(DispatchKeySet ks) const;
   // Returns true if kernel_ has entry for a particular key.
   bool hasKernelForDispatchKey(DispatchKey k) const;
+  // Returns all the operator tags added at the time of registration
+  const std::vector<at::Tag>& getTags() const;
 
 private:
 
   OperatorName name_;
   c10::optional<AnnotatedSchema> schema_;
-
+  #ifndef C10_MOBILE
```

Comment:
nit: would be good to have a comment here that explains why we're doing this (binary size).